packages/payments-plugin/src/mollie/mollie.controller.ts

```diff
@@ -20,8 +19,10 @@ export class MollieController {
             return Logger.warn(' Ignoring incoming webhook, because it has no body.id.', loggerCtx);
         }
         try {
+            // We need to construct a RequestContext based on the channelToken,
+            // because this is an incoming webhook, not a graphql request with a valid Ctx
+            const ctx = await this.createContext(channelToken);
             await this.mollieService.handleMollieStatusUpdate(ctx, {
-                channelToken,
```

Comment:
Removed channelToken to prevent confusion between `ctx.channel.token` and the `channelToken` metadata in the Mollie webhook